### ISSUE TITLE
Member function on string literal causing two problems

### ISSUE THREAD

**Issue by michaelalandawson** [softwareCobbler/cfc#10]
Two problems are detected with the following valid code:

````#"2022-11-08".lsDateFormat("full")#````

The first error is at the "." character.

[{
	"resource": "/c:/[redacted]/wwwroot/news-and-events/index.cfm",
	"owner": "_generated_diagnostic_collection_name_#1",
	"severity": 8,
	"message": "Unterminated hash-wrapped expression.",
	"source": "cfls",
	"startLineNumber": 113,
	"startColumn": 63,
	"endLineNumber": 113,
	"endColumn": 64
}]

The second error is at the ending "#" character.

[{
	"resource": "/c:/[redacted]/wwwroot/news-and-events/index.cfm",
	"owner": "_generated_diagnostic_collection_name_#1",
	"severity": 8,
	"message": "Expected an expression.",
	"source": "cfls",
	"startLineNumber": 113,
	"startColumn": 85,
	"endLineNumber": 113,
	"endColumn": 86
}]

Thanks!

**Comment by softwareCobbler** [comment]
should be fixed on 1.0.44, released now.

Some differing behavior between engines here:
```
" foo ".trim()    // acf ok, lucee ok
" foo "["trim"]() // acf no, lucee ok
v = "trim"
" foo "[v]()      // acf no, lucee ok
```

**Comment by michaelalandawson** [comment]
Wonderful!  Thanks for the update.
